src/domains/chart/components/lib-charts/dygraph/utils.ts

```diff
@@ -37,14 +37,24 @@ export const getDataForFakeStacked = (
   ]
 })
 
+const isPercentage = (unit: string) => unit === "percentage"
```

Comment:
`/(percent|%)/.test(unit)`

Reply:
I am trying to be as close to the business requirement as possible. I prefer exactly the word in order to avoid cases like 'percentile' or 'x/percent'